.gitignore

```diff
@@ -355,3 +358,7 @@ lib/ojdbc.jar
 
 # do not ignore JabRef icons (they are ignored by the macos setting above)
 !src/main/java/org/jabref/gui/icon
+
+# Ignore some eclipse annotation stuff
+/.apt_generated/
+/.apt_generated_tests/
```

Comment:
The first one is generated by https://www.gitignore.io/api/eclipse

The second one refs https://github.com/github/gitignore/pull/3193